appveyor.yml

```diff
@@ -0,0 +1,23 @@
+environment:
+  global:
+    CIBW_SKIP: pp36-macosx_x86_64 pp37-macosx_x86_64
```

Comment:
I think you could add `cp27* cp35* pp*` (or similar syntax) here, and that would make builds faster by avoiding those